### PR TITLE
feat: user preferences

### PR DESCRIPTION
**Pull request by Meierschlumpf**
<br/>
<div align="center">
  
  <h3>Homarr</h3>
</div>

**Thank you for your contribution. Please ensure that your pull request meets the following pull request:**

- [x] Builds without warnings or errors (``pnpm buid``, autofix with ``pnpm format:fix``)
- [x] Pull request targets ``dev`` branch
- [x] Commits follow the [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] No shorthand variable names are used (eg. ``x``, ``y``, ``i`` or any abbrevation)

## Screenshots

image

image

image

image

image

image

### CODE REVIEW

File: apps/nextjs/src/app/[locale]/manage/users/[userId]/_profile-avatar-form.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  const t = useI18n();
+  const tManageAvatar = useScopedI18n("user.action.manageAvatar");
+
+  const handleAvatarChange = useCallback(
+    async (file: File | null) => {
+      if (!file) {
+        return;
+      }
+
+      const base64Url = await fileToBase64(file);
```
Comment (manuel-rw) [inline apps/nextjs/src/app/[locale]/manage/users/[userId]/_profile-avatar-form.tsx:37]: Rename with ``async`` suffix

File: apps/nextjs/src/app/[locale]/manage/users/[userId]/_profile-avatar-form.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+              leftSection={<IconPencil size={18} stroke={1.5} />}
+            >
+              {t("common.action.edit")}
+            </Button>
+          </UnstyledButton>
+        </Menu.Target>
+        <Menu.Dropdown>
+          <FileButton
+            onChange={handleAvatarChange}
+            accept="image/png,image/jpeg"
```
Comment (manuel-rw) [inline apps/nextjs/src/app/[locale]/manage/users/[userId]/_profile-avatar-form.tsx:139]: What about ``WebP``? Do we allow ``GiF``?

**Review by manuel-rw — CHANGES_REQUESTED** [review]

File: apps/nextjs/src/app/[locale]/manage/users/[userId]/_profile-avatar-form.tsx (hunk 1 above)
Comment (Meierschlumpf) [reply apps/nextjs/src/app/[locale]/manage/users/[userId]/_profile-avatar-form.tsx]: Done

File: apps/nextjs/src/app/[locale]/manage/users/[userId]/_profile-avatar-form.tsx (hunk 2 above)
Comment (Meierschlumpf) [reply apps/nextjs/src/app/[locale]/manage/users/[userId]/_profile-avatar-form.tsx]: Added both

**Review by manuel-rw — APPROVED** [review]

**Pull request merged**